crates/telio-relay/src/derp/proto.rs

```diff
@@ -122,6 +122,9 @@ enum FrameType {
     Ping = 0x12,
     /// 8 byte payload, the contents of the ping being replied to
     Pong = 0x13,
+    /// controll message for communication with derp. Since these messages are not
+    /// for communication with other peers through derp, they don't contain public_key
+    ControlMessage = 0x14,
```

Comment:
This may need a longer discussion together with @Jauler.
There is kindof two approaches here:
- Current one where we basically have 3 types on channels with relay:
    - Node to Relay, 
    - Node to Relay to Node
    - Node to Node
   And inner telio-proto type specifies the actual type, but it's a bit weird, because it's not that clear from telio-proto side witch now goes where.
   We already have some confusion with Node to Node vs Node to Relay to Node, and now we are adding more.
   
 - Second approch would be to add Request, Response directly here, without extra wrapping layer, it would be a bit cleaner from protocol side, but I think it will complicate message passing to other telio components :/

Reply:
I did use the second approach at the start, but it looked pretty messy as the only distinction for the new derp message types was that they don't have public keys, so I wanted to add some generic message type for Node to Relay messages

Reply:
Would it make sense to use one channel from telio-relay for `Node -> Relay -> Node` messages and another for `Node -> Relay` messages? 

`Node -> Node` case does not go through telio-relay, so I am thinking, that probably there is no need to distinguish it into its own separate type.